Automattic/wp-calypso
# Update restricted imports for Calypso.

This is a follow-up to [a comment in another PR](https://github.com/Automattic/wp-calypso/pull/35783#discussion_r318521481).

It adds two changes to the ESLint config:

- Add `gridicons` restricted import. We want to migrate away from the "naked" import, and use `components/gridicon` explicitly, instead of relying on the webpack alias forever.
- Rewrite `wpcalypso/import-no-redux-combine-reducers` as a restricted import.

~~Note that this does not delete the `wpcalypso/import-no-redux-combine-reducers` rule code, to avoid breaking compatibility for consumers of the `eslint-plugin-wpcalypso` package.~~ After some feedback from @sirreal, I removed the rule.

#### Testing instructions

* Ensure that any file with `import Gridicon from 'gridicons';` is marked as having a lint error.
* Add `import { combineReducers } from 'redux';` to any file, and ensure that it's marked as having a lint error.
